src/components/add-note-card/add-note-card.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import styles from './add-note-card.module.scss';
+import { Button } from '../button/button';
+import { PlusIcon } from '@radix-ui/react-icons';
+import { useNavigate, Link } from 'react-router-dom';
+
+export interface AddNoteCardProps {
+    className?: string;
+}
+
+export const AddNoteCard = ({ className }: AddNoteCardProps) => {
+    const navigate = useNavigate();
+    return (
+        <div className={classNames(styles.root, className)}>
+            <Button onClick={() => navigate('/add-note')}>
```

Comment:
We can try to create another component.
`LinkButton` with typed `to` property as a union.

Then anyone would be able to use it via AddPanel and configure where it navigates with Props panel.

<img width="286" alt="Screenshot 2024-05-14 at 14 39 20" src="https://github.com/wixplosives/NotesApp/assets/124175587/21c2228a-2f96-40a3-a472-6da1c3a55e47">

Reply:
But `to` will be dynamic in some cases, like a page to edit a note, `/notes/:id` somethings like this